src/workspace/files/.stylelintrc.template

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "stylelint-config-standard",
```

Comment:
I was wondering whether we could make this `"extends": "@ppwcode/angular-schematics"` somehow so that we can push changes to the defaults as a new version of this package.

Will look into this in a different issue: https://github.com/peopleware/angular-ppwcode-schematics/issues/16